pom.xml

```diff
@@ -620,19 +620,6 @@
         <scope>test</scope>
       </dependency>
       <dependency>
-        <groupId>org.easymock</groupId>
-        <artifactId>easymockclassextension</artifactId>
-        <version>3.1</version>
-        <scope>test</scope>
-      </dependency>
-      <!-- Needed by cglib which is needed by easymock. -->
```

Comment:
It should be fine. I added it because easymock was throwing a fit for me (but not on jenkins). I actually had an alternate patch at the time to replace easymock with mockito, should have sent that for review...